src/components/SearchPanel/SearchPanel.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+import SearchStyle from './SearchPanel.module.scss';
+import { SearchState } from './Search.types';
+import { PropsWithChildren, useState } from 'react';
+import { getDefaultFormValues } from './Search.utils';
+import Image from 'next/image';
+
+export function SearchPanel({
+  setSearchQuery,
+  placeholder,
+}: {
+  setSearchQuery: (query: string) => void;
+  placeholder?: string;
+}) {
+  const handleSubmit = (event: React.SyntheticEvent) => {
+    event.preventDefault();
+    const encodedSearch = encodeURI(formState.request);
+    setSearchQuery(encodedSearch);
+    setFormState(getDefaultFormValues);
+  };
+  const [formState, setFormState] = useState<SearchState>(
```

Comment:
What do you think of 
`const [inputValue, setInputValue] = useState('')`? Isn't it simpler?